webserver/templates/user/import.html

```diff
@@ -37,6 +37,22 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
         </div>
      {% else %}
      <p>
+        <strong>Notes about the Last.fm import process:</strong>
+     </p>
+     <p>
+        The Last.fm importer manually steps through your listen history and imports the listens
```

Comment:
Wanted, but this section is getting big. How about I add them to the second step?